cli/src/pcluster/config/cluster_config.py

```diff
@@ -1206,10 +1206,6 @@ def shared_mount_dir_list(self):
         if self.shared_storage:
             for storage in self.shared_storage:
                 mount_dir_list.append(storage.mount_dir)
-
-        if self.head_node.local_storage.ephemeral_volume:
-            mount_dir_list.append(self.head_node.local_storage.ephemeral_volume.mount_dir)
-
         return mount_dir_list
 
     @property
```

Comment:
The property below contains the mount dir of ephemeral volume of head node. So the old code count the mount dir of head node ephemeral volume twice, generating the duplicate mount dir error